### PR TITLE
Update README.md with my github link and brief description

### LINKED ISSUE #7: Add Annie team member info to frontend README
Please open a PR which modifies the  frontend `README` to add info about yourself at the top:

At the very top of the doc, please add an h1 heading for `Team Members`

And then add your information as a bullet point:

- **myFirstName** (@myGithubUsername) someInfoAboutMe 

Formatting:
* add bold to your name
* add a link to your username pointing to your github profile page

You can see more info about Github's markdown syntax in context from the existing `README` content, as well as in the documentation [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

### PR DESCRIPTION
**Pull request by anniethewebcoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - **Jillian** \([@catbus00](https://github.com/catbus00)\) is a student studying React, but showing Backend Team some support!
 - **Mohammad** ([@msrezaie](https://github.com/msrezaie)), A MERN stack aficionado
 - **Victoria** ([@Victoria240](https://github.com/Victoria240)), I'm a foodie 😋
-
+- **Annie** ([@anniethewebcoder](https://github.com/anniethewebcoder) is a studying React and Ruby on Rails at Code the Dream.
```
Comment (dargaCode) [inline README.md]: Looking good, just a small typo fix:

```suggestion
- **Annie** ([@anniethewebcoder](https://github.com/anniethewebcoder)) is studying React and Ruby on Rails at Code the Dream.
```

**Review by dargaCode — CHANGES_REQUESTED** [review]
Thanks Annie! Just need to fix up the parentheses and small typo and you'll be good to go!

File: README.md (hunk 1 above)
Comment (anniethewebcoder) [reply README.md]: Got it, thanks!

**Comment by anniethewebcoder** [comment]
I fixed a grammatical error as per Darga's suggestion.

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 - **Jillian** \([@catbus00](https://github.com/catbus00)\) is a student studying React, but showing Backend Team some support!
 - **Mohammad** ([@msrezaie](https://github.com/msrezaie)), A MERN stack aficionado
 - **Victoria** ([@Victoria240](https://github.com/Victoria240)), I'm a foodie 😋
-
+- **Annie** ([@anniethewebcoder](https://github.com/anniethewebcoder) is studying React and Ruby on Rails at Code the Dream.
```
Comment (dargaCode) [inline README.md]: Sorry you're still missing the extra closing perenthesis after your usename!

```suggestion
- **Annie** ([@anniethewebcoder](https://github.com/anniethewebcoder)) is studying React and Ruby on Rails at Code the Dream.
```

**Review by dargaCode — CHANGES_REQUESTED** [review]
One more small fix please =D

**Comment by anniethewebcoder** [comment]
I added ")"

**Review by dargaCode — APPROVED** [review]
Great!

**Pull request merged**